### ISSUE TITLE
VU meters in Standard and Normal layout modes start out horizontal

### ISSUE THREAD

**Issue by dakhubgit** [jamulussoftware/jamulus#1122]
In Normal layout mode, local and strip VU meters start out horizontal.  The same is true for the Compact layout mode's local meter (but not the strip meters).  The more complex VU meters in Fancy mode are fine, however.

**To Reproduce**
Switch on Compact or Normal mode when connected to some server.  Slowly increase the amount of local input and wait for the VU meters to appear.

**Expected behavior**
VU meters should always be vertical

**Screenshots**
See [this video](https://youtu.be/_J3pIUo2A9k) for a demonstration

**Operating system**
Ubuntu Linux 20.10.

**Version of Jamulus**
3.6.2dev-d993e155
**Additional context**
~~~
||/ Name              Version       Architecture Description
+++-=================-=============-============-=================================
ii  qt5-default:amd64 5.14.2+dfsg-6 amd64        Qt 5 development defaults package
~~~

**Comment by dcorson-ticino-com** [comment]
I can not reproduce this on Win10.

**Comment by hoffie** [comment]
Can't reproduce either with Jamulus 3.6.2dev-00b7f4e8.

Qt 5.15.2 (x86_64-little_endian-lp64 shared (dynamic) release build; by GCC 10.2.0) on "wayland" 
OS: Arch Linux [linux version 5.10.9-arch1-1]

As you've got a video, I guess this is reproducible for you?
Do you know whether this problem existed in previous versions of Jamulus?
Your version is listed as d993e155 and I don't find this commit id in the main git. Do you run some special build?

**Comment by dakhubgit** [comment]
Christian Hoffmann <notifications@github.com> writes:

> Can't reproduce either with Jamulus 3.6.2dev-00b7f4e8.
>
> Qt 5.15.2 (x86_64-little_endian-lp64 shared (dynamic) release build;
> by GCC 10.2.0) on "wayland"
> OS: Arch Linux [linux version 5.10.9-arch1-1]
>
> As you've got a video, I guess this is reproducible for you?
> Do you know whether this problem existed in previous versions of Jamulus?
> Your version is listed as d993e155 and I don't find this commit id in
> the main git. Do you run some special build?

Well, oops, this was while developing the MIDI controller code.  The
commits in question are completely unrelated to this issue, the next
upstream commit would be

commit 35f677b00f516bbe7225ce9d9bebf7402b1d3dad
Merge: a92db0ea 82b899cb
Author: Peter L Jones <pljones@users.noreply.github.com>
Date:   Wed Feb 10 21:51:15 2021 +0000

    Merge pull request #968 from dcorson-ticino-com/pr957-server-recording
    
    feature-req925 recording indicator

I've never so far had this work under Ubuntu (I started with Jamulus
October or November), and it's been always Ubuntu 20.10 (and
consequently the corresponding Qt5 version) I've been working with in
connection with Jamulus.  It's pretty likely related to the given Qt
version and possibly its distribution- (or even user-)depending
configuration.

This is an UbuntuStudio setup, and I only started using KDE since
UbuntuStudio now switched to it from Xfce4.  I cannot vouch for some
Qt/KDE interactions being involved here or not.

If really necessary, I may try to start the binaries under different
desktop systems to see whether KDE is involved here.

-- 
David Kastrup

**Comment by hoffie** [comment]
> If really necessary, I may try to start the binaries under different
desktop systems to see whether KDE is involved here.

This would be very helpful to find out if it is a general issue or something specific to the UbuntuStudio environment. :)

**Comment by hoffie** [comment]
Any updates here?
Maybe it's worth trying a recent build from master as there have been several UI changes. Also, Ubuntu 21.04 has recently been released which might improve things. I've also noticed that that the pan wheel looks different than on my machine, so this might be related to a general difference in the relevant theme?

**Comment by dakhubgit** [comment]
In the meantime the UI has changed, I have updated to Ubuntu Hirsute (21.04), _and_ I had to change my Desktop back from KDE (the Ubuntustudio default) to XFCE4 because my GPU was no longer stable.  There was a Qt development library update I had to follow through with as well (Hirsute appears to have a version of Jamulus of its own, so this was a matter of apt-get build-dep jamulus).  It would appear that the problem is gone.

Now the million dollar question: should I try to figure out which of the numerous changes might have been responsible?  Or should we just chalk this off as "well, this was reported only by this one person"?

I probably should at least try the distribution version of Jamulus (which likely has the older UI) since chances are that any problems in there are seen by more people than just myself?

**Comment by hoffie** [comment]
I think it would be interesting to find out what it was. To keep the required effort low, maybe try just the distribution version which sounds simple to do?

**Comment by dakhubgit** [comment]
> I think it would be interesting to find out what it was. To keep the required effort low, maybe try just the distribution version which sounds simple to do?

Except that my laptop microphone refused to work for whatever reason and it took me quite a bit of time until I found some configuration where it would do anything...  At any rate, the distribution version appears to work fine.

I suspect this might have had to do something with the Qt (development?) version on 20.10.  It is also worth noting that when I was running Xfce4 on the older version, the menus in the main window (Help/Profile/Whatever) were not there.  I needed to run KDE to see them.

That is not the case on Hirsute's precompiled Jamulus client: the menus are there in Xfce4 just fine.  So my guess is that the Qt library version (at least the development variant) coming with 20.10 was broken, or something on my system broke them.
